src/screens/SendTokens.js

```diff
@@ -186,7 +182,16 @@ class SendTokens extends React.Component {
         arr.push(Buffer.from(signatures[i]));
       }
       this.sendTransaction.prepareTxFrom(arr);
-      this.setState({ ledgerStep: 1, ledgerModalTitle: t`Sending transaction` });
+      this.setState({
+        ledgerStep: 1,
+      }, () => {
+        this.context.showModal(MODAL_TYPES.ALERT, {
```

Comment:
Why are you showing this modal here? I don't see anything showing this modal before.

Reply:
Because we could change the modal body using `setState` before, now we can't 

So what this is doing is removing the old modal and replacing it by a new one with the `renderAlertBody` rendered with `ledgerStep` set to 1